<launch>

    <node pkg="bno055_ros" type="plot_imu_data_node" name="plot_imu_data" clear_params="true" output="screen">
        <rosparam file="$(find bno055_ros)/config/plot_imu_data.yaml" />
    </node>

</launch>
